cfpb/cfgov-refresh
# Improve print styling and strip current path from in-page anchor links

Addressing https://JIRA/browse/INTAKE-4238

## Changes

- When printing a cf.gov page:
  - Link `href`s are now output after the link text
  - In-page anchor links are unstyled
  - Several unhelpful-in-print elements are no longer displayed:
    - Footer navigation links
    - Email signup and RSS subscription blocks
    - Video player play buttons
  - Video players will print with "Please visit this webpage to view the above video." appended
- Python link parsing middleware updated to strip the current path from in-page anchor links. See the commit message on c001112 for details.

## Removals

- Unused data-qa-hooks were removed from the RSS subscription block template (along with the one browser test shared object file that referenced them, but was never used in any actual 
tests)


## Testing

1. Pull branch
1. `gulp styles`
1. Load http://localhost:8000/about-us/blog/guide-coronavirus-mortgage-relief-options/
1. Print preview (or add `.print` to the `html` element and view in a small screen layout)
1. Verify the changes described above


## Checklist


## Testing checklist

### Browsers

- [ ] Internet Explorer 9 and 11
